ddprof-exporter/Cargo.toml

```diff
@@ -15,6 +15,11 @@ bytes = "1.0"
 chrono = "0.4"
 futures = "0.3"
 http = "0.2"
+lazy_static = "1.4.0"
```

Comment:
This should be just `"1.4"` for consistency, unless you are aware of a bug in some newer version in which case it should be commented :)